engine/xml-config/src/test/resources/JaxbJobWriterTest-file4.xml

```diff
@@ -16,6 +16,9 @@
     <transformation>
         <transformer>
             <descriptor ref="Email standardizer"/>
+            <metadata-properties>
+                <property name="source">local</property>
```

Comment:
OK. Property "source" is not filled when the value is "local", so only remote components have this metadata property and its value is the server name.